### ISSUE TITLE
Backgrounding the app using deactivateAppForDuration does not work with Xcode12b1

### ISSUE THREAD

**Issue by dostrander** [kif-framework/KIF#1155]
In Xcode12 it seems that UIAutomation has been removed which is the mechanism we use to background the application.

We need to look into another mechanism to background the application for Xcode12.

**Comment by dostrander** [comment]
One possible idea for this is to look in the XCTAutomationSupport to help with this. In Xcode 12/iOS 14 class-dump is broken so this makes it harder and I have not found any APIs in the Xcode11 version of apple that may work for our use cases. We may need to do some disassembling to find out how XCUI does this

**Comment by kenji21** [comment]
For what I looked, the XCUI is a separate processus, which can tell SpringBoard (using xpc) to launch app, and have it going in background

**Comment by dostrander** [comment]
Yeah it is, however how they call it is the question we need to figure out. If they use a framework that we can dynamically link similar to how we dynamically link `UIAutomation` we should be able to call it and do the same thing, even in process AFAIK
